jina/docker/hubio.py

```diff
@@ -75,6 +76,75 @@ def new(self) -> None:
         except click.exceptions.Abort:
             self.logger.info('nothing is created, bye!')
 
+    def login(self) -> None:
+        """Login using Github Device flow to allow push access to Jina Hub Registry"""
+        with resource_stream('jina', '/'.join(('resources', 'hubapi.yml'))) as fp:
+            hubapi_yml = yaml.load(fp)
+        
+        client_id = hubapi_yml['github']['client_id']
```

Comment:
Minor thing, would it make sense to keep all these `constants` listed somewhere instead of hardcoded inside the code?

Reply:
I've tried keeping the values in a yml file under `resources`